components/input/OTP/OTPInput.tsx

```diff
@@ -61,18 +65,31 @@ const OTPInput = React.forwardRef<InputRef, OTPInputProps>((props, ref) => {
 
   // ========================= Render =========================
   return (
-    <Input
-      type={mask === true ? 'password' : 'text'}
-      {...restProps}
```

Comment:
```suggestion
      {...restProps}
      type={mask ? 'password' : 'text'}
```

Is this change enough?

Reply:
@afc163 I did that in my previous pull request but @zombieJ suggested [User passed props type should be single source of truth. We should not override this](https://github.com/ant-design/ant-design/pull/53415#discussion_r2030611325)